phnt/include/ntexapi.h

```diff
@@ -22,6 +22,7 @@
 #define _NTEXAPI_H
 
 #include <ntkeapi.h>
+#include <ntimage.h>
```

Comment:
Why does this also need to be imported?

Reply:
Because of `PIMAGE_EXPORT_DIRECTORY` used in `SYSTEM_GDI_DRIVER_INFORMATION` structure.

Reply:
Hmm, now I noticed that `ntimage.h` is only part of WDK.
And PHNT's `ntexapi.h` is used also in ProcessHacker (non-WDK) project.
So this change will make PH not compilable without WDK.

Will that be a problem?
Or WDK is required either way?

Reply:
Eventually we can use forward declaration instead of `ntheader.h` include:
`typedef struct _IMAGE_EXPORT_DIRECTORY * PIMAGE_EXPORT_DIRECTORY;`

What do you think?

Reply:
We don't use that structure so I changed the type